### DISCUSSION TITLE
Verification Assistance for GitHub Education Benefits as a Chinese Student without an Educational Email

### DISCUSSION THREAD

**Discussion by shikangyao** [community/community#113390] [GitHub Education]
### Hi everyone,

Dear GitHub Education Team,

I hope this email finds you well. I am a Chinese student eager to leverage GitHub for learning and collaboration purposes. I understand that to access certain educational benefits on GitHub, verification through an educational email is typically required.

Unfortunately, my institution does not provide educational email addresses to its students, and I am therefore unable to proceed with the standard verification process. I would greatly appreciate it if you could advise me on how to proceed in such a scenario.

I assure you that all the information I provide is authentic and accurate. I am sincerely interested in using GitHub to enhance my learning experience and contribute to open-source projects.

Could you please suggest an alternative method for verification that does not rely on an educational email? I am willing to provide any additional information or documentation that may be required for verification purposes.

Thank you for your time and consideration. I look forward to your response and the opportunity to become a part of the vibrant GitHub community.

Best regards,

Shikangyao

**Comment by samus-aran** [comment]
Hi @shikangyao 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [I don't have a school-issued email address, what other official proof of enrollment is accepted?](https://github.com/orgs/community/discussions/111352#i-dont-have-a-school-issued-email-address)
- [You appear to not be near any campus location for the school you have selected.](https://github.com/orgs/community/discussions/111352#not-near-any-campus-location-for-selected-school)
- [I need to translate my evidence. What can I do?](https://github.com/orgs/community/discussions/111352#translate-my-evidence)

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍
